docs/source/cli-options.md

```diff
@@ -104,6 +104,31 @@ Run the uninstaller in silent mode from its original location:
 > cmd.exe /c start /wait "C:\Program Files\my_app\uninstaller.exe" /S _?=C:\Program Files\my_app
 ```
 
+:::{admonition} Stream redirection in EXE installers
+:class: info
+
+When run in `/S` mode, EXE installers will output some minimal info to the console when invoked
+like this:
+
+```batch
+> cmd.exe /c start /wait my_installer.exe /S
```

Comment:
`cmd.exe /c` is not necessary. You could also consider adding PowerShell since this is the default profile in the Windows Terminal

```
Start-Process -FilePath my_installer.exe -ArgumentList "/S" -NoNewWindow -Wait
```

Reply:
Nice, you can even use `-RedirectStandardOutput out.txt` to correctly redirect things!

Reply:
Added this to the documentation.

Reply:
It looks like you replaced the documentation. I don't see the harm showing both `cmd.exe` and PowerShell.

Reply:
Added tabs with both now.